chemprop/cli/convert.py

```diff
@@ -5,44 +5,60 @@
 
 from chemprop.cli.utils import Subcommand
 from chemprop.utils.v1_to_v2 import convert_model_file_v1_to_v2
+from chemprop.utils.v2_0_to_v2_1 import convert_model_file_v2_0_to_v2_1
```

Comment:
Since v2.2 will soon be released, and some users might want to convert models from 2.0 to 2.2, how about generalizing the notation (e.g. `v2_0_to_v2_x` or `v2_0_to_current`)?